src/dso_api/dynamic_api/filters/parser.py

```diff
@@ -33,12 +33,13 @@
     "like": str,  # e.g. uri is parsed as string for like.
 }
 
-MULTI_VALUE_LOOKUPS = {"not": operator.and_}
-SPLIT_VALUE_LOOKUPS = {"in"}
+MULTI_VALUE_LOOKUPS = {"not", "in"}  # which lookups may hold multiple values
+SPLIT_VALUE_LOOKUPS = {"in", "contains"}  # which lookups use comma-separated in the querystring
+LOOKUP_MERGE_OPERATORS = {"not": operator.and_}  # which ones should be merged for the QuerySet
```

Comment:
It this what you mean by the comments?

```suggestion
MULTI_VALUE_LOOKUPS = {"not", "in"}  # lookups that may occur multiple times in a single request
SPLIT_VALUE_LOOKUPS = {"in", "contains"}  # lookups that take comma-separated value lists
LOOKUP_MERGE_OPERATORS = {"not": operator.and_}  # which ones should be merged for the QuerySet
```